pyrobosim/pyrobosim/navigation/rrt.py

```diff
@@ -365,7 +364,7 @@ def try_connect_until(self, graph, n_curr, n_tgt):
                 else:
                     return False, n_curr
             else:
-                # If not using RRT-Connect, we only get one chance to connect.
+                # If not using RRTConnect, we only get one chance to connect.
```

Comment:
I am learning a lot about hyphens in planning algorithm names from this PR!

Reply:
I actually also noticed this, but thought it would be too pedantic to change it. Seems you have surpassed me :)

Reply:
I am rapidly-exploring pedantry.